lib/src/widgets/credit_card.dart

```diff
@@ -208,6 +217,24 @@ class _CreditCardState extends State<CreditCard> {
     );
   }
 
+  void _onChangeCvc(value) => _cardData.cvc = value ?? '';
```

Comment:
Missing types for all parameters below
Should be 
`void _onChangeCvc(String? value)`
`void _onChangeName(String? value)`
etc